executors/dart_web/pubspec.yaml

```diff
@@ -8,7 +8,11 @@ environment:
 
 # Add regular dependencies here.
 dependencies:
-  intl4x: ^0.4.0
+  intl4x:
```

Comment:
To be changed back to a regular pub.dev dependency once https://github.com/dart-lang/i18n/pull/702 is merged and published.